src/config/config.h

```diff
@@ -94,6 +94,19 @@ bool cfgParameterAllowed(void);
 // Command parameters, if any
 const StringList *cfgCommandParam(void);
 
+/***********************************************************************************************************************************
+Option Group Functions
+***********************************************************************************************************************************/
+// Are any options in the group with the specified index valid and set?
+bool cfgOptionGroupIndexTest(ConfigOptionGroup groupId, unsigned int index);
```

Comment:
Maybe just a clarification in the comment regarding what "valid" really means - maybe:
`Are any options in the group with the specified index valid (has been set by the user) and set?`

or something to that affect because the default value is "valid" but because the value source is cfgSourceDefault then this function will return false.

Reply:
Updated.